unattended-upgrade

```diff
@@ -504,15 +504,39 @@ def upgrade_in_minimal_steps(cache,            # type: apt.Cache
     # double check any changes we do
     allowed_origins = get_allowed_origins()
 
+    # pre-calculate set sizes to process sets which are expected to be smaller
+    # earlier
+    upgrade_set_sizes = {}
```

Comment:
Won't this need a mypy type annotation?

Reply:
I have not used mypy on this project but now trying it it reports several issues which I'll resolve in the next PR.
I'm also adding an annotation for this variable.

Reply:
Thanks! I added mypy hints a while ago, I thought I also added a corresponding test that is run in travis. Let me check what is going on there.